en/docs/InstallAndSetup/PerfromanceTuningAndTestResults/tuning-performance.md

```diff
@@ -81,41 +81,54 @@ In a clustered environment, the entity expansion limit has no dependency on the
--   [Tuning data-agent parameters](#TuningPerformance-Tuningdata-agentparameters)
+-   [Timeout configurations for an API call](#timeout-configurations-for-an-api-call)
+-   [General APIM-level recommendations](#general-apim-level-recommendations)
+-   [Registry indexing configurations](#registry-indexing-configurations)
+-   [Tuning data-agent parameters](#tuning-data-agent-parameters)
 
 ##### Timeout configurations for an API call
 
 The following diagram shows the communication/network paths that occur when an API is called. The timeout configurations for each network call are explained below.
-![]({{base_path}}/assets/attachments/103333648/103333650.png)
+
+![](../../../../assets/attachments/103333648/103333650.png)
+
+!!! info
+    Gateway to Keymanager network call to validate token only happens with the OAuth token. This network call does not happen for JWT tokens . From WSO2 API Manager 3.0.0 onwards, JWT tokens are the default token type for applications. Since JWT tokens are self-contained access tokens, Keymanager is not needed to validate the token. The token is validated from the gateway.
+    
 -   **Key validation**
     Key validation occurs via a Servlet HTTP call and the connection timeout can be configured by changing the following configuration details in the `<API-M_HOME>/repository/conf/axis2/axis2_client.xml` file. All timeout values are in milliseconds.
 
+    !!! danger
+        Cannot configure using deployment.toml
     ``` xml
-            <transportSender name="https" class="org.apache.axis2.transport.http.CommonsHTTPTransportSender">
-            <parameter name="SO_TIMEOUT">60000</parameter>
-            <parameter name="CONNECTION_TIMEOUT">60000</parameter>
-            </transportSender>
+    <transportSender name="https" class="org.apache.axis2.transport.http.CommonsHTTPTransportSender">
+        <parameter name="SO_TIMEOUT">60000</parameter>
+        <parameter name="CONNECTION_TIMEOUT">60000</parameter>
+    </transportSender>
     ```
 
     If the Key Manager caching is enabled, the calls between the API Gateway and Key Manager are cached. As a result, the Key Manager is not invoked for each API call.
 
 -   **Client call API Gateway + API Gateway call Backend**
-    For backend communication, the API Manager uses PassThrough transport. This is configured in the `<API-M_HOME>/repository/conf/passthru-http.properties` file. For more information, see [Configuring passthru-http.properties](https://docs.wso2.com/display/EI620/Configuring+passthru-http.properties) in the EI documentation.
+    For backend communication, the API Manager uses PassThrough transport. This is configured in the `<API-M_HOME>/repository/conf/deployment.toml` file. For more information, see [Configuring passthru properties](https://docs.wso2.com/display/EI620/Configuring+passthru-http.properties) in the EI documentation. To configure socket timeout value, add following to the `deployment.toml`
```

Comment:
Why are we pointing to EI 6.2.0 when EI 6.5.0 is the lastest EI version that is compatible with APIM 3.0.0? The link needs to be updated.

Reply:
fixed